RELEASE_PROCESS.md

```diff
@@ -27,43 +27,123 @@ In order to release, you will need the following accounts/permissions:
 
 ## THE PROCESS
 
-### Informing everyone of a pending release
-
-- [ ] Announce your intention to drive the release to #cft-announce and #chef-server on slack.
-
 ### Getting the build to be released into current with a minor/major version bump
 
-- Update the version file to the version to be released if minor version needs to be bumped rather than just the patch.
-- Update the release notes with the version, date and context.
-- Run all the tasks for update from FrequentTasks.doc
-- Make sure the omnibus build is into current channel.
-  (This is triggered by expeditor once the build and tests in buildkite go through ok once a commit is merged to master)
-- Make sure the habitat builds are passing.
-- For updating just the patch verion refer the the section below on #Preparing for the release
+- Create a new branch from the latest master branch. [QUESTION: or do we simply push a commit directly to master?]
+- Update the release notes with the version, date and context.  
+  https://github.com/chef/chef-server/wiki/Pending-Release-Notes
+- Run all the tasks for 'Updating Ruby Gems' from dev-docs/FrequentTasks.md.  Note that some of this might have already been done by dependabot.
+- Push branch, create PR.
+- Bump the release version:
+    - Apply the label _Expeditor: Bump Version Minor_ or _Expeditor: Bump Version Major_ to your PR to bump the version number of the release candidate, as applicable. [QUESTION: can BOTH labels be applied at once?]
```

Comment:
Perhaps but we will usually bump to the next minor or major. Since versions are major.minor.patch; bumping to minor sets patch to 0 and bumping the major version will update minor.patch to 0.0